pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/NamespacesTest.java

```diff
@@ -2106,4 +2106,63 @@ public void testDeleteTopicPolicyWhenDeleteSystemTopic() throws Exception {
                 .getTopicPoliciesBypassCacheAsync(TopicName.get(systemTopic)).get(5, TimeUnit.SECONDS);
         assertNull(topicPolicies);
     }
+
+    @Test
+    public void testCreateNamespacesWithPolicy() throws Exception {
+        try {
+            namespaces.createNamespace(this.testTenant, "other-colo", "my-namespace",
+                    new Policies());
+            fail("should have failed");
+        } catch (RestException e) {
+            // Ok, cluster doesn't exist
+        }
+
+        List<NamespaceName> nsnames = new ArrayList<>();
+        nsnames.add(NamespaceName.get(this.testTenant, "use", "create-namespace-1"));
+        nsnames.add(NamespaceName.get(this.testTenant, "use", "create-namespace-2"));
+        nsnames.add(NamespaceName.get(this.testTenant, "usc", "create-other-namespace-1"));
+        createTestNamespaces(nsnames, new Policies());
+
+        try {
+            namespaces.createNamespace(this.testTenant, "use", "create-namespace-1",
+                    new Policies());
+            fail("should have failed");
+        } catch (RestException e) {
+            // Ok, namespace already exists
```

Comment:
can we assert the response error code in all the cases in this test case?

Reply:
Added asserts for the response error codes